### DISCUSSION TITLE
Copilot Activation Error

### DISCUSSION THREAD

**Discussion by benhurbemey** [community/community#12945] [Copilot Conversations]
I'm having copilot activation problem after reinstalling my windows operating system.
The error message is as shown in this image. 
What should I do?
Thank you.

![github coppilot](https://user-images.githubusercontent.com/19516045/158656207-931dd8fe-1faf-4668-ae69-b8a353470e77.png)

**Comment by hsenag** [comment]
Hi, thanks for the report. Can I check if you're using the latest VSCode?
